### PR TITLE
whitespace cleanup

### PR DESCRIPTION
**Pull request by Karsten1987**
fixes https://github.com/eclipse/iceoryx/issues/5

as per title. There are a bunch of trailing whitespaces in iceoryx at the moment.

Signed-off-by: Knese Karsten <karsten.knese@us.bosch.com>

### CODE REVIEW

**Comment by mossmaurice** [comment]
Hi,

thanks for this PR! Especially trailing white spaces are a pain. Could you create an issue and link it with this PR (adding it to the commit message would be great as well) so that one can trace all the changes back to a Github issue?

Cheers!

**Comment by Karsten1987** [comment]
@mossmaurice @dkroenke Any update on this?

File: iceoryx_posh/source/mepoo/mepoo_config.cpp [hunk 1]
```diff
@@ -50,8 +50,8 @@ MePooConfig& MePooConfig::setDefaults() noexcept
     m_mempoolConfig.push_back({1024 * 1024 * 2, 20});
     m_mempoolConfig.push_back({1024 * 1024 * 4, 10});
     m_mempoolConfig.push_back({1024 * 1024 * 8, 10});
-    m_mempoolConfig.push_back({1024 * 1024 * 16, 5});
```
[the remark this answers is not in the document]
Comment (Karsten1987) [reply iceoryx_posh/source/mepoo/mepoo_config.cpp]: legacy changes for presentation.

File: iceoryx_examples/icedelivery/a_typed_api.hpp [hunk 2]
```diff
@@ -29,7 +29,7 @@ using SamplePtr = std::unique_ptr<TopicType, SampleDeleter<TopicType>>;
 /// @brief A typed publisher that takes the topic type as template argument
 /// A RAII pattern is used with offering the the publisher in c'tor and stop offering in d'tor
 /// This class has the limitation that the topic type is a fixed size data structure.
-/// I.e. we can get the size of memory to allcoate with the sizeof() operation. 
+/// I.e. we can get the size of memory to allcoate with the sizeof() operation.
```
[the remark this answers is not in the document]
Comment (Karsten1987) [reply iceoryx_examples/icedelivery/a_typed_api.hpp]: addressed in https://github.com/eclipse/iceoryx/pull/2/commits/a204c131c7a05aaececbb74ece4855cdb38b515b

**Comment by Karsten1987** [comment]
Please find the latest commit which addresses your comments.

**Pull request merged**